Dockerfile

```
@@ -0,0 +1,5 @@
+FROM openjdk:8-alpine
+VOLUME /tmp
+EXPOSE 8081
+COPY ./target/CSS-Backend-1.0-SNAPSHOT-jar-with-dependencies.jar app.jar
```

Better move these VOLUME and EXPOSE "annotations" down, it's a common practice as far as I know.

```suggestion
COPY ./target/CSS-Backend-1.0-SNAPSHOT-jar-with-dependencies.jar app.jar
VOLUME /tmp
EXPOSE 8081
```

---

Ok, will do in a second